packages/server/src/queue/queue.entity.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  BaseEntity,
+  ManyToOne,
+  JoinColumn,
+  OneToMany,
+} from 'typeorm';
+import { CourseModel } from '../course/course.entity';
+import { QuestionModel } from '../question/question.entity';
+import { UserPartial, OpenQuestionStatus } from '@template/common';
+import { Exclude, Expose } from 'class-transformer';
+
+@Entity('queue_model')
+export class QueueModel extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne((type) => CourseModel, (course) => course.queues)
+  @JoinColumn({ name: 'courseId' })
+  course: CourseModel;
+
+  @Column({ nullable: true })
+  @Exclude()
+  courseId: number;
+
+  @Column('text')
+  room: string;
+
+  @OneToMany((type) => QuestionModel, (qm) => qm.queue)
+  @Exclude()
+  questions: QuestionModel[];
+
+  @Expose()
+  get staffList(): UserPartial[] {
```

Comment:
nit: can we just leave a TODO here so we know that this is temp wh ile we figure out OH -> queues?